models/vocoders/diffusion/diffwave/diffwave.py

```diff
@@ -26,6 +26,11 @@ def silu(x):
     return x * torch.sigmoid(x)
 
 
+# This model code is adopted from DiffWave/model.py under the Apache License
```

Comment:
Change line29-31 into the header comments.

Reply:
Finished